src/Snackbar/snackbar.component.js

```diff
@@ -16,6 +16,8 @@ const styles = theme => ({
     flexWrap: 'nowrap',
   },
   snackbar: {
+    position: 'absolute',
+    bottom: '50px',
```

Comment:
I don't think this works in the case when it isn't deployed on a PNADA does it? The extra bar is only added then, we might have to modify this in future to take the 50px from a settings file.